.github/updatecli.d/bump-go-microsoft-version.sh

```diff
@@ -29,12 +29,10 @@ echo "Update go version ${GO_RELEASE_VERSION}"
 
 find "go" -type f -name Dockerfile.tmpl -print0 |
     while IFS= read -r -d '' line; do
-        ${SED} -E -e "s#(ARG GOLANG_VERSION)=[0-9]+\.[0-9]+(\.[0-9]+)?#\1=${MAJOR_MINOR_PATCH_VERSION}#g" "$line"
```

Comment:
golang-version is managed in the bump-golanag.yml.

This is just about the security when the same patch version exists but the Microsoft golang version has changed